From 5d3d7a9bbbb9463bc190730e1b5ff0f52d83ee81 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Edwin=20T=C3=B6r=C3=B6k?= <edwin.torok@cloud.com>
Date: Tue, 5 Mar 2024 09:55:23 +0000
Subject: [PATCH] CA-389506: fix platform:nested_virt typo
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Nested virt is not a supported feature yet, but when moving the setting
from Xen to toolstack as part of the Xen-4.17 update I typoed the name of the setting.

It was platform/nested-virt in xenguest.patch before

Fixes: 664de7608d ("Xen-4.15+: CDF_NESTED_VIRT")
Signed-off-by: Edwin Török <edwin.torok@cloud.com>
---
 ocaml/xenopsd/xc/domain.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ocaml/xenopsd/xc/domain.ml b/ocaml/xenopsd/xc/domain.ml
index bd17e5d28..935027aab 100644
--- a/ocaml/xenopsd/xc/domain.ml
+++ b/ocaml/xenopsd/xc/domain.ml
@@ -351,7 +351,7 @@ let make ~xc ~xs vm_info vcpus domain_config uuid final_uuid no_sharept =
   if iommu then
     assert_capability CAP_DirectIO ~on_error:(fun () -> "IOMMU unavailable") ;
   let nested_virt =
-    get_platform_key ~key:"nested_virt" ~default:false require_hvm
+    get_platform_key ~key:"nested-virt" ~default:false require_hvm
   in
   let vpmu = get_platform_key ~key:"vpmu" ~default:false (fun _ -> Ok ()) in
 
-- 
2.39.5

